src/common/base/src/vec_ext.rs

```diff
@@ -24,4 +31,32 @@ impl<T> VecExt<T> for Vec<T> {
         let pos = self.iter().position(|x| x == item)?;
         Some(self.remove(pos))
     }
+
+    #[inline]
```

Comment:
Better to have some unit tests for the new function in the base crate.

Reply:
add quicktests